app/components/ConfirmTrade.tsx

```diff
@@ -200,3 +284,21 @@ export const ConfirmTrade = ({
     </Dialog>
   );
 };
+
+const Spinner = (props: SVGProps<SVGSVGElement>) => (
+  <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" {...props}>
+    <circle
+      className="opacity-25"
+      cx="12"
+      cy="12"
+      r="10"
+      stroke="currentColor"
+      stroke-width="4"
+    ></circle>
+    <path
+      className="opacity-75"
+      fill="currentColor"
+      d="M4 12a8 8 0 018-8V0C5.373 0 0 5.373 0 12h4zm2 5.291A7.962 7.962 0 014 12H0c0 3.042 1.135 5.824 3 7.938l3-2.647z"
+    ></path>
+  </svg>
+);
```

Comment:
1. Should we add this icon to our icon set on `swapr-ui`?

2. This is a good example of a  component I'm in favor of moving this spinner out to it's own file. why? 
Because we'll probably will use some kind of loading in the future, and I'm afraid, by being hidden inside this file, we don't remember we already have a spinner.

Reply:
I think we should add it to `swapr-ui`. But in the meantime, while we don't add it I think it can live here.

Reply:
can you create a ticket to add spinner icon to `swapr-ui`?

It can live here in the meanwhile.

Reply:
Here it is https://linear.app/swaprhq/issue/SWUI-44/add-spinner-icon